code/obj/railing.dm

```diff
@@ -0,0 +1,344 @@
+// this is such a dumb joke
+/obj/meme
```

Comment:
I'm too lazy to check the icon files but maybe this could have a slightly more descriptive name? Or not! This is just a suggestion.